service/src/main/java/se/inera/statistics/service/warehouse/NationellData.java

```diff
@@ -18,40 +18,19 @@
  */
 package se.inera.statistics.service.warehouse;
 
+import static se.inera.statistics.service.warehouse.ResponseUtil.filterCutoff;
+
 import java.time.LocalDate;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Projektet har tidigare haft regler mot asterisk-importer. Jag hittar inte dessa längre, men jag har svårt att tro att det blivit någon förändring (det handlar nog bara om att jag inte hittar det helt enkelt). Det går att ställa in intellij så att den inte automatiskt ändrar till asterisk-import för att själv slippa hålla koll på detta.

Reply:
Ändrat tillbaka imports.